src/applications/vaos/tests/e2e/page-objects/ClinicChoicePageObject.js

```diff
@@ -6,10 +6,15 @@ class ClinicChoicePageObject extends PageObject {
     return this;
   }
 
-  assertUrl() {
+  assertUrl({ isCovid = false } = {}) {
     cy.url().should('include', '/clinic', { timeout: 5000 });
     cy.axeCheckBestPractice();
 
+    this.clickNextButton();
+    if (!isCovid) {
```

Comment:
Covid uses an older ClinicChoicePage design that uses raw radio widgets instead of VaRadioField. This means that its alerts are not nested in the shadow DOM and are therefore covered in the unit tests. We only need these assertions in the non-covid clinic choice pages.